boards/airbourne/airbourne_board.h

```diff
@@ -88,6 +95,10 @@ class AirbourneBoard : public Board
   LED led2_;
   LED led1_;
   M25P16 flash_;
+  AnalogDigitalConverter battery_adc_;
+  AnalogPin voltage_pin_;
+  AnalogPin current_pin_;
+  BatteryMonitor battery_monitor_;
```

Comment:
I've corrected this in the battery monitor branch, which hasn't been merged yet. It moves the pins into the BatteryMonitor class, and has an init struct for the battery monitor. The ADC stays out because while the battery monitor uses it, it does not require exclusive use.